src/main/java/com/coveo/spillway/storage/utils/CacheSynchronization.java

```diff
@@ -56,6 +56,8 @@ public CacheSynchronization(InMemoryStorage cache, LimitUsageStorage storage) {
 
   @Override
   public void run() {
+    logger.info("Synchronizing spillway storage cache.");
```

Comment:
Not sure it's relevant to log here and it might spam the logs on frequent update schedules.

Reply:
You're probably right. I'll remove it.